src/lib/components/inputs/InputSelect.svelte

```diff
@@ -0,0 +1,249 @@
+            ? "Enter email ID or name of team member."
+            : ""}
+          autocomplete="off"
+          autocapitalize="none"
+          style="outline:none;border:none;flex-grow:1; background:transparent;"
+          bind:value={currentEmailEntered}
+          on:input={() => {
+            filterUser();
+          }}
+          on:focus={() => {
+            isOpen = true;
+          }}
+          on:blur={() => {
+            setTimeout(() => {
+              isOpen = false;
+            }, 500);
+          }}
+          class="input-container mt-2 sparrow-fs-12"
+        />
+      </div>
+    </div>
+
+    {#if isOpen}
+      <div
+        class="d-none select-data p-1 rounded"
+        class:select-active={isOpen}
+        transition:slide={{ duration: 100 }}
+      >
+        {#each data as user, index}
+          <div
+            class="d-flex w-100 px-2 py-2 highlight"
+            on:click={() => {
+              handleEmailOnAdd(user.email);
+              filterUser();
+            }}
+          >
+            <div class="d-flex tile w-100 rounded align-items-center">
+              <div class="info d-flex align-items-center">
+                <div
+                  class="icon d-flex align-items-center justify-content-center"
+                >
+                  <span>{user.name[0].toUpperCase()}</span>
+                </div>
+                <div class="name px-2">
+                  <span style="font-size:12px;" class="text-whiteColor"
+                    >{user.name || ""}</span
+                  ><br />
+                  <span style="font-size:12px;" class="text-textColor"
+                    >{user.email}</span
+                  >
+                </div>
+              </div>
+            </div>
+          </div>
+        {/each}
+        {#if !data?.length}
+          <p
+            class="m-0 ps-2 p-0 pt-1 pb-1 text-whiteColor"
+            style="font-size: 12px;"
+          >
+            {"No members found!"}
+          </p>
+        {/if}
+      </div>
+    {/if}
+  </div>
+</div>
+
+<style>
+  .email-container {
+    display: flex;
+    flex-wrap: wrap;
+    background-color: transparent;
+    border: 1px solid;
+    padding: 3px 8px 3px 8px;
+    border: 1px solid var(--border-color);
+    max-height: 100px;
+    overflow-y: auto;
+  }
+  input {
+    margin-bottom: 4px !important;
+    margin-top: 4px !important;
+  }
+  #input-email {
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    gap: 5px;
+  }
+  .select-data {
+    background-color: rgba(0, 0, 0, 0.7);
+    color: white;
+    position: absolute;
+    bottom: 0px;
+    left: 0;
+    right: 0;
+    transform: translateY(100%);
+    border: 1px solid rgb(44, 44, 44);
+    max-height: 200px;
+    overflow-y: auto;
+    -webkit-backdrop-filter: blur(10px); /* For some older versions of Safari */
+    backdrop-filter: blur(10px);
+  }
+  .select-data p {
+    font-size: 12px;
```

Comment:
I think we have previously used fs class to set 12 px. Lets use that.

Reply:
resolved